JASP-Engine/JASP/R/commonsummarystatsttestbayesian.R

```diff
@@ -202,7 +202,7 @@
   if (is.null(container)) {
     container <- createJaspContainer()
     # add dependencies for main table (i.e., when does it have to recompute values for the main table)
-    container$dependOn(c("tStatistic", "n1Size", "n2Size", "hypothesis", "bayesFactorType",            # standard entries
```

Comment:
The robustness check does need a dependency on the `bayesFactorType`.
The flipping of axes I spoke of earlier was about this plot apparently ;)

Reply:
@TimKDJ Can you fix this issue?

Reply:
@akashrajkn I am not so sure anymore what we have to do with this PR. Tim is ill. Should we merge it?

Reply:
@boutinb I need to make a change in this. Can you wait till tomorrow?

Reply:
OK

Reply:
@akashrajkn Can you have a look today? Then we can close this issue.

Reply:
@akashrajkn have you looked into this now?